docs/how-to/hip_porting_guide.rst

```diff
@@ -0,0 +1,649 @@
+.. meta::
+  :description: This chapter presents how to port CUDA source code to HIP.
+  :keywords: AMD, ROCm, HIP, CUDA, porting, port
+
+################################################################################
+HIP porting guide
+################################################################################
+
+HIP is designed to ease the porting of existing CUDA code into the HIP
+environment. This page describes the available tools and provides practical
+suggestions on how to port CUDA code and work through common issues.
+
+********************************************************************************
+Porting a CUDA Project
+********************************************************************************
+
+Mixing HIP and CUDA code results in valid CUDA code. This enables users to
+incrementally port CUDA to HIP, and still compile and test the code during the
+transition.
+
+The only notable exception is ``hipError_t``, which is not just an alias to
+``cudaError_t``. In these cases HIP provides functions to convert between the
+error code spaces:
+
+:cpp:func:`hipErrorToCudaError`
+:cpp:func:`hipCUDAErrorTohipError`
+:cpp:func:`hipCUResultTohipError`
+
```

Comment:
there is one more to add here:
hipErrorToCUResult

Reply:
Added

Reply:
Thanks.